README.md

```diff
@@ -48,7 +48,10 @@ Prerequisites for building Apache Fluss:
 ```bash
 git clone https://github.com/apache/fluss.git
 cd fluss
+# in case of java 11
 ./mvnw clean package -DskipTests
+# or in case of java 8
+./mvnw clean package -DskipTests -Pjava8
```

Comment:
```
./mvnw clean package -DskipTests
```
is not true anymore for java8 and fails like 
```
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal com.diffplug.spotless:spotless-maven-plugin:2.27.1:check (spotless-check) on project fluss: Execution spotless-check of goal com.diffplug.spotless:spotless-maven-plugin:2.27.1:check failed: You are running Spotless on JVM 8, which limits you to google-java-format 1.7.
[ERROR] Upgrade your JVM or try google-java-format alternatives:
[ERROR] - Version 1.7 requires JVM 8+
[ERROR] - Version 1.15.0 requires JVM 11+

```

to make it working it is required to put profile explicitly